docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -14,289 +14,324 @@ need to implement it on your own. Don't worry, it's easy!
+If we run
 
 ::
 
-    from coalib.settings.Setting import typed_list
+    $ pylint --help
 
-``typed_list(item_type)`` is a function that converts the given input
-(which the user will pass to the Bear as a setting) into a list of
-items and afterwards will apply a conversion to type ``item_type`` to each
-item in the list (you can also use basic types like ``int``, ``bool``, etc.
-see :doc:`Writing Bears <Writing_Bears>`)
-Next, we have to add our setting as a parameter for the ``run()`` method
-of our bear.
-We will give the param a sugestive name like ``htmllint_ignore``.
+We can see that there is a ``--rcfile`` option which lets us specify a
+configuration file for Pylint. Let's add that functionality to our bear.
 
 ::
 
-    def run(self,
-            filename,
-            file,
-            htmllint_ignore: typed_list(str)=[]):
-        '''
-        Checks the code with `html_lint.py` on each file separately.
-
-        :param htmllint_ignore: List of checkers to ignore.
-        '''
+    import os
 
-.. note::
-
-    The documentation of the param is parsed by *coala* and it will be used
-    as help to the user for that specific setting.
+    from coalib.bearlib.abstractions.Linter import linter
+    from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
-The last thing we need to do is join the strings in the ``html_ignore``,
-append them to ``--disable=`` and add it as an argument. There are alot
-of ways of doing that.
+    @linter(executable='pylint',
+            output_format='regex',
+            output_regex=r'L(?P<line>\d+)C(?P<column>\d+): '
+                         r'(?P<message>(?P<severity>[WFECRI]).*)',
+            severity_map={'W': RESULT_SEVERITY.NORMAL,
+                          'F': RESULT_SEVERITY.MAJOR,
+                          'E': RESULT_SEVERITY.MAJOR,
+                          'C': RESULT_SEVERITY.NORMAL,
+                          'R': RESULT_SEVERITY.NORMAL,
+                          'I': RESULT_SEVERITY.INFO})
+    class PylintTutorialBear:
+        @staticmethod
+        def create_arguments(filename, file, config_file,
+                             pylint_rcfile: str=os.devnull):
+            return ('--msg_template="L{line}C{column}: {msg_id} - {msg}"',
+                    '--reports=n', '--rcfile=' + pylint_rcfile, filename)
+
+Just adding the needed parameter to the ``create_arguments`` signature
+suffices, like you would do for other bears inside ``run``! Additional
+parameters are automatically queried from the coafile. Let's add also some
+documentation:
 
 ::
 
-    ignore = ','.join(part.strip() for part in htmllint_ignore)
-    self.arguments = '--disable=' + ignore
-    return self.lint(filename)
+    @linter(...)
+    class PylintTutorialBear:
+        """
+        Lints your Python files!
 
-Right place for '{filename}'
-----------------------------
+        Checks for codings standards (like well-formed variable names), detects
```

Comment:
we had some discussion about this, if you want to change it trigger a new one. We agreed on imperative tense for doc comments and will soon have a bear for it.